src/coffee/sync/utils/base.coffee

```diff
@@ -17,8 +17,9 @@ class BaseUtils
         includeValueOnMove: false  # value of items moved is not included in deltas
       textDiff:
         # if value to diff has a bigger length, a text diffing algorithm is used
+        # disable the text diffing algorithm by passing an unreachable minimum length
         # https://github.com/benjamine/jsondiffpatch/blob/master/docs/deltas.md#text-diffs
-        minLength: 300
+        minLength: Infinity
```

Comment:
@emmenko try to run only the test `'should build the action update'` and remove this Infinity part or put it back to 300. It will use google-diff-match-patch algorithm for longer text, which will produce the error.